src/overlays/actors/ovl_Item_Ocarina/z_item_ocarina.c

```diff
@@ -7,19 +7,21 @@
 #include "z_item_ocarina.h"
 #include "assets/scenes/overworld/spot00/spot00_scene.h"
 
+//ocarina object for the zelda escaping with impa cutscene and flashback in light arrow cutscene
```

Comment:
You can move this above, where `Description: Ocarina of Time` is
(possibly polishing up the punctuation :blush: )